frontend/src/middleware/auth/auth-guard.js

```diff
@@ -14,7 +14,6 @@ function isGoingToIntegrationsPage(to) {
  * It uses the PermissionChecker to validate if:
  * - User is authenticated, and both currentTenant & currentUser exist within our store (if not, redirects to /auth/signin)
  * - Email of that user is verified (if not, redirects to /auth/email-unverified)
```

Comment:
I think we can remove this comment since we longer handle it here, wdyt?

Also, what's the intended action if the email of the user is not verified? are we handling this check somewhere else?

Reply:
Great point! In linux this is kind of deprecated. Since SSO this logic is currently not handled in our app anymore that's why the pages were removed. So I'm also gonna delete this comment to avoid confusions

Reply:
Cool, let's merge it 🚀